### PR TITLE
Search best practices part 1 - blogpost

### PR DESCRIPTION
**Pull request by acverrette**

### CODE REVIEW

File: _posts/2017-09-06-search-best-practices-1.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+# Conclusion
+
+Overall you need to remember that your search box has to be prominent and easy to locate. In some use cases, minimalist search boxes might work fine, but before you choose this design, try to test it with real users to see if they really can find the search box easily. 
+There is no perfect recipe that will work with every website, but if you follow those best practices, it will surely increase the search engagement on your website and will certainly result in a higher conversion rate.
+
+
+
+<sup>1</sup> NIELSEN, J., SHADE, A. E-Commerce User Experience. _Nielsen Norman Group_. 5(3),148p. 
+
+<sup>2</sup> SHERWIN, K. (2014). Placeholders in Form Fields Are Harmful. _Nielsen Norman Group_. Retrieve from https://www.nngroup.com/articles/form-design-placeholders/
```
Comment (amoreauCoveo) [inline _posts/2017-09-06-search-best-practices-1.md]: Retrieved*

File: _posts/2017-09-06-search-best-practices-1.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+tags: [Search, UX]
+
+author:
+  name: Anne-Catherine Verrette
+  bio: UX Specialist
+  image: acverrette.jpg
+---
+
+Organizations are becoming more and more aware of the importance of offering a good search experience on their website. From increases in purchases on ecommerce websites to getting fewer cases on their support and community websites, companies are realizing that helping users find what they want faster and more easily results in a great overall user experience.
```
Comment (erocheleau) [inline _posts/2017-09-06-search-best-practices-1.md]: >from *an* increase in purchases?

Comment (erocheleau) [inline _posts/2017-09-06-search-best-practices-1.md]: ecommerce or e-commerce?

Comment (erocheleau) [inline _posts/2017-09-06-search-best-practices-1.md]: > great overall user experience

great *er* overall user experience

File: _posts/2017-09-06-search-best-practices-1.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+In this series of blog posts, I will explain why having a good search experience is vital for any website. This blog post will tell you about the importance of the search box.
+
+<!-- more -->
+
+In order to deliver a good search experience, users have to be able to find the search box. It may seem obvious, but too many websites fail to make their search box prominent and easy to find. There are many reasons that can explain that. One of them is that, in the past years, there has been a design trend to hide the search box behind a link or a nice magnifying glass icon.
+
+However, while they please the eye and match the look and feel of the website, those search boxes can do more harm than you might think. For example, did you know that if your search box is hidden behind a link, users will assume that you are trying to hide your search because it is poor?<sup>1</sup> To prevent that, always make sure that your search box is visible and prominent by following these tips.
```
Comment (erocheleau) [inline _posts/2017-09-06-search-best-practices-1.md]: > those search boxes can do more harm than...

Do you mean _those search icons_ ?

File: _posts/2017-09-06-search-best-practices-1.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+# First rule of good search: finding the search box
+
+> The good old open field is good for a reason
+
+When users want to perform a search, they are used to scan the page looking for an open search field, not a link or an icon. Even if it might seem obvious for us, a lot of users don’t recognize the magnifying glass icon as the search tool.
```
Comment (erocheleau) [inline _posts/2017-09-06-search-best-practices-1.md]: > used to scan the page

used to scanning the page

**Pull request merged**